### ISSUE TITLE
Add precise CPU time accounting for tasks.

### ISSUE THREAD

**Issue by prioux** [aces/cbrain#565]
CBRAIN tasks now all report their amount of CPU time taken in a small 4-lines report at the bottom of their qsub log:

```
CBRAIN Task CPU Times Start
0m0.001s 0m0.002s
917m11.667s 6m5.579s
CBRAIN Task CPU Times End
```

I am not sure which of these numbers is user vs system times, but recording them would be good.

I propose a new table with columns ( user_id, group_id, bourreau_id, tool_id, tool_config_id, run_number, task_id, task_type, systime, usertime ), inserted every time a task is completed. The row would stay in the DB even if any of the other models are deleted (the task, the user, the group etc), and that is why I duplicate many of the values of the task object itself.

The interface to this table would be able to show reports with sums on these attributes; it is likely we can use directly the built-in report maker in fact (but with a twist, that we don't expand the full table, which will accumulate many rows).

**Comment by prioux** [comment]
Related to #571 (might even be duplicate)

**Comment by natacha-beck** [comment]
@prioux Does this issue and issue #571 can be closed now? With your new report.

**Comment by prioux** [comment]
All implemented in #865
